lxd/device/disk.go

```diff
@@ -798,7 +798,10 @@ func (d *disk) startVM() (*deviceConfig.RunConfig, error) {
 				err = func() error {
 					sockPath, pidPath := d.vmVirtiofsdPaths()
 					logPath := filepath.Join(d.inst.LogPath(), fmt.Sprintf("disk.%s.log", d.name))
-					os.Remove(logPath) // Remove old log if needed.
+					err = os.Remove(logPath) // Remove old log if needed.
```

Comment:
I think we can get away with just having the error ignored here as vmVirtiofsdPaths will fail to start if it cannot create new log file.